### DISCUSSION TITLE
Bros whats up im student, does anyone know what the yolo v7 formula is?

### DISCUSSION THREAD

**Discussion by NoBullShiet** [community/community#33101] [Other Feature Feedback, Questions, & Ideas]
Hi , i have a nice idea but i I need the mathematical formula Yolo V7  , If anyone knows how I can get it or where to start I would be very grateful

Thank you everyone read this

**Comment by cosmicqbit** [comment]
Hi buddy, Mark this as answered if it helps: https://viso.ai/deep-learning/yolov7-guide/
